src/Sulu/Bundle/MediaBundle/Media/DispositionType/DispositionTypeService.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\DispositionType;
+
+use Sulu\Bundle\MediaBundle\Entity\FileVersion;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+
+/**
+ * Service handling media disposition type.
+ */
+class DispositionTypeService implements DispositionTypeServiceInterface
```

Comment:
naming here is little bit strange as all in symfony is mostly a service I would call it `DispositionTypeResolver`